RevenueCatUI/Templates/Example1Template.swift

```diff
@@ -4,48 +4,82 @@ import SwiftUI
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 struct Example1Template: TemplateViewType {
 
-    private let packages: [Package]
-    private let localization: PaywallData.LocalizedConfiguration
+    private let package: Package
+    private let localization: ProcessedLocalizedConfiguration
     private let configuration: PaywallData.Configuration
 
     init(
         packages: [Package],
         localization: PaywallData.LocalizedConfiguration,
         configuration: PaywallData.Configuration
     ) {
-        self.packages = packages
-        self.localization = localization
+        // The RC SDK ensures this when constructing Offerings
+        precondition(!packages.isEmpty)
```

Comment:
What does `precondition` do? 🤔

Reply:
Also crash if it’s not true. But in a more obvious way than getting an element from an empty array, just being more explicit.
But per the comment, the SDK ensures you can’t construct this.